osu.Game/Overlays/Comments/DrawableComment.cs

```diff
@@ -403,6 +413,12 @@ private void deleteCommentRequest()
             api.Queue(request);
         }
 
+        private void copyUrl()
+        {
+            host.GetClipboard()?.SetText($@"https://osu.ppy.sh/comments/{Comment.Id}");
```

Comment:
Bit of a minor point but hardcoding `https://osu.ppy.sh/` as the root URL here is incorrect. Should be reading [`APIAccess.APIEndpointUrl`](https://github.com/ppy/osu/blob/master/osu.Game/Online/API/APIAccess.cs#L37) instead.

This matters in debug / when using the development server since in that case the root URL is `https://dev.ppy.sh/`.